### ISSUE TITLE
Midpoint Review - Design Feedback - DS Logon Grandfathering / Inherited Proofing

### ISSUE THREAD

**Issue by allison0034** [department-of-veterans-affairs/va.gov-team#55063]
## VFS acceptance criteria
- [ ] Let Platform know when the **Must** feedback has been incorporated 
- [ ] Leave any comments for feedback you decide _not_ to take
- [ ] VFS team closes the ticket

## Thoughts/questions
- Dont forget to submit the new link style and close button to experimental design.

## Feedback
- Practice areas will document their feedback following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback).

**Must:**
- On the modal, the font size looks a bit small. Make sure the H1 is 20px and the body text is 16px. Change the upper right circle with the X to primary blue color.

**Should:**
- Get rid of any extra clicks or logins if you can.
- Once user testing is complete, align the content and experience as much as possible.

**Consider:**

## Platform directions
- Remove the governance-team label
- Update "Issue Title"
- Link to collab cycle Request epic
- Add your feedback
- Remove the CC-Request label (only if you created the ticket using the New Issue link from the Collaboration Cycle Reviews board)
- Add assignees based on collab cycle touchpoint
  - **Design Intent**: VFS designer, VFS PM (optional), yourself (optional)
  - **Midpoint Review**: VFS PM, yourself (optional)
  - **Staging Review QA Only**: VFS PM, yourself (optional)
